cli/config/configuration-fc.toml.in

```diff
@@ -19,6 +19,9 @@ path = "@FCPATH@"
 # This is disabled by default as additional setup is required
 # for this feature today.
 #jailer_path = "@FCJAILERPATH@"
+# jailer_chroot_base_dir represents the base folder where chroot jails are built.
+# The default is /run/vc.
```

Comment:
it'd be great if you can add a *"heads up"* and mention that in some systems `/run` is mounted with `noexec` and this is incompatible with the jailer, etc, etc...

Reply:
okay~